contracts/multisig-prover/src/encoding/evm.rs

```diff
@@ -58,6 +58,28 @@ impl TryFrom<WorkerSet> for Command {
     }
 }
 
+impl TryInto<Operators> for WorkerSet {
+    type Error = ContractError;
+    fn try_into(self) -> Result<Operators, Self::Error> {
+        let mut operators: Vec<(HexBinary, Uint256)> = self
+            .signers
+            .iter()
+            .map(|s| {
+                (
+                    evm_address((&s.pub_key).into())
+                        .expect("couldn't convert pubkey to evm address"),
+                    s.weight,
+                )
+            })
+            .collect();
+        operators.sort_by_key(|op| op.0.clone());
```

Comment:
Do you mean the `Operators` instead of the `WorkerSet`? the sort should be based on the evm address which is not part of the worker set. Should we have a separate refactor task for this one?